code/datums/components/label.dm

```diff
@@ -22,11 +22,12 @@
 	apply_label()
 
 /datum/component/label/RegisterWithParent()
-	RegisterSignal(parent, COMSIG_PARENT_ATTACKBY, PROC_REF(OnAttackby))
+	RegisterSignal(parent, COMSIG_PARENT_ATTACKBY, PROC_REF(onAttackBy))
 	RegisterSignal(parent, COMSIG_PARENT_EXAMINE, PROC_REF(Examine))
```

Comment:
Just change these two to be `on_attack_by` and `on_examine()` or something, and this should be good to go.